src/components/TintedImage/index.tsx

```diff
@@ -8,10 +9,20 @@ export interface TintedImageProps {
 }
 
 export function TintedImage({ className, src }: TintedImageProps) {
+  const [errored, setErrored] = useState(false);
+
   return (
     <div className={className}>
       <div className={styles.overlay} />
-      <Image alt="" className={styles.image} fill src={src} />
+      {errored ? null : (
+        <Image
+          alt=""
+          className={styles.image}
+          fill
+          onError={() => setErrored(true)}
+          src={src}
+        />
+      )}
```

Comment:
This way, if the speaker doesn't have an image under `public/speakers/` (e.g. how halfstack.com/new-york is right now), we don't get the browser default 404 image icon.